src/doc-templates/index.md.jinja2

```diff
@@ -1,7 +1,19 @@
-# {{ schema.name }}
+# ![NMDC_logo.png](images/NMDC_logo.png)
+# National Microbiome Data Collaborative Schema Documentation
 
 {{ schema.description }}
 
+This schema is organized into three key modules:
+
+* A core set of elements for representing data values represented in LinkML format (https://w3id.org/linkml/) consisting
+of a set classes, slots, types, and enumerations that are used to define the structure of the NMDC schema.
+
+* A subset of the MIxS schema developed by the Genomic Standards Consortium, that is used to describe the
+environmental context of samples.
+
+* An annotation schema that provides a mechanism for annotating data with additional information, such as provenance,
+quality, and other metadata that is not part of the core data model.
+
```

Comment:
This isn't really true any more. Right now there are more than three modules (disregarding whatever "key" means), but I would like to switch to just two: one NMDC module and one GSV MIxS module